src/style.css

```diff
@@ -1,3 +1,73 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  height: 60px;
+  display: flex;
+  background-color: #ffff;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.logo_link {
+  height: 40px;
+  width: 40px;
+  display: block;
+  margin-left: 25px;
+}
+
+.nav_link {
+  height: 15px;
+  color: #000;
+  position: relative;
+  font-weight: bold;
+  font-family: Roboto, sans-serif;
+  text-wrap: nowrap;
+  font-size: 12px;
+  text-decoration: none;
+  text-transform: uppercase;
+  display: block;
+  padding: 23px 0;
+  margin-right: 20px;
+}
+
+.nav_link-1 {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  display: block;
+  content: ' ';
+  position: absolute;
+  left: 0;
+  right: 0;
+  width: 100%;
+  height: 4px;
+  bottom: 0;
+  background-color: #00acdc;
+  border-radius: 8px;
+}
+
+.nav_item {
+  list-style: none;
```

Comment:
Avoid using tag names for styling. Instead, use classes to apply styles. This makes your CSS more flexible and maintainable. For example, instead of styling 'ul' directly, use '.nav_list' to apply the 'list-style: none;' property.